openerp/addons/base/res/res_partner.py

```diff
@@ -339,16 +322,11 @@ def copy(self, default=None):
         return super(res_partner, self).copy(default)
 
     @api.multi
-    def onchange_type(self, is_company):
+    def onchange_type(self, company_type):
         value = {'title': False}
```

Comment:
no more required this method. both field 'use_parent_address' and 'domain' removed.